app/com/linkedin/drelephant/tony/heuristics/TaskGPUHeuristic.java

```diff
@@ -75,6 +75,13 @@ public HeuristicResult apply(TonyApplicationData data) {
       details.add(new HeuristicResultDetails("Number of " + taskType + " tasks",
           Integer.toString(taskMap.get(taskType).size())));
 
+      // get number of GPU resource requested, if any
+      Integer numGPUsRequested = conf.getInt(TonyConfigurationKeys.getResourceKey(taskType, "gpus"), 0);
```

Comment:
Please use `int` to avoid auto-boxing/unboxing.

You can also use com.linkedin.tony.Constants.GPUS instead of "gpus".